app/src/main/java/edu/ucandroid/weathernotice/ui/main/MainViewModel.kt

```diff
@@ -34,10 +34,6 @@ class MainViewModel : ViewModel() {
         weatherLocations = weatherService.fetchAllWeather()
     }
 
-    fun fetchAllWeather() {
```

Comment:
Simply just removing technical debt here, really has no impact on the project/code itself, I just noticed that this method was already defined in the same class and this particular method was never being called.

Reply:
I believe that one of the member plan still using it for a another use in another branch. That this time I do think was a use for this in another branch that load data into a recycler view.